### ISSUE TITLE
Accept uncobbled Deepslate and Netherack as fuel

### ISSUE THREAD

**Issue by immortius** [immortius/chunkbychunk#120]

**Comment by JaxTheWolf** [comment]
I agree this should be implemented. Netherrack is plentiful, but it sadly can't be used for the world forge.

**Comment by Insane96** [comment]
Agree on deeplate but disagree on Netherrack, it's too easy to obtain.

**Comment by JaxTheWolf** [comment]
> Agree on deeplate but disagree on Netherrack, it's too easy to obtain.

That's true, but I think it could be handled the same way as dirt is.
